eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Cert.hs

```diff
@@ -125,8 +126,8 @@ certTransition = do
     ConwayTxCertPool poolCert -> do
       newPState <- trans @(EraRule "POOL" era) $ TRC (PoolEnv slot pp, certPState, poolCert)
       pure $ cState {certPState = newPState}
-    ConwayTxCertCommittee committeeCert -> do
-      newVState <- trans @(EraRule "VDEL" era) $ TRC (VDelEnv, certVState, committeeCert)
+    ConwayTxCertCommittee vDelCert -> do
```

Comment:
I think we can also rename the `ConwayTxCertCommittee` constructor.

Reply:
Can I do this along with the other the renames together in a follow up PR?